packages/db/prisma/seed.ts

```diff
@@ -10,7 +10,7 @@ async function main() {
     update: {
       sessions: {
         update: {
-          where: { id: "23" },
+          where: { sessionId: "23" },
```

Comment:
I disambiguated id's in our schema for clarity, you'll see lots of files where this is the only change